#!/usr/bin/env python3

from django.core.exceptions import ValidationError
